app/README.md

```diff
@@ -0,0 +1 @@
+# Sample Application
```

Comment:
@svrnm could you please provide a couple of instructions on how to run it? 

e..g via the docker-compose and/or k8s?